lib/fluent/plugin/in_windows_eventlog.rb

```diff
@@ -55,6 +59,20 @@ def configure(conf)
                           else
                             method(:no_encode_record)
                           end
+      if @store_pos
+        unless system_config.root_dir
```

Comment:
If `persistent true`, storage plugin should raise configuration errors if any paths are unavailable.
So this check is not needed, and moreover, this check prohibits users to specify `<storage> path </storage>` configuration sections manually.